### PR TITLE
Added Projects navbar

### LINKED ISSUE #12: Navbar - Project Pages
Add Navbar to projects page and ensure the links works.

### PR DESCRIPTION
**Pull request by NathanBlackman**
Added a navbar to link to the other pages.

### CODE REVIEW

File: projects.html [hunk 1]
```diff
@@ -16,7 +16,32 @@
...
+                        </button>
+                        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                            <li class="nav-item">
+                              <a class="nav-link" href="index.html">Overview</a>
+                            </li>
+                            <li class="nav-item">
+                              <a class="nav-link" href="repositories.html">Repositories</a>
+                            </li>
+                            <li class="nav-item">
```
Comment (TrinityChristiana) [inline projects.html:33]: Switch the order of this one and the Projects. Order of the Navabr should be Overview, Repositories, Projects, and Packages.

**Review by pmpurcell — DISMISSED** [review]
Good!

**Review by TrinityChristiana — APPROVED** [review]
LGTM!

**Review by austincparker — APPROVED** [review]
it's goooood

**Pull request merged**